examples/mxnet_imagenet_resnet50.py

```diff
@@ -303,7 +300,7 @@ def main():
                         'lr_scheduler': lr_sched}
     if args.dtype == 'float16':
         optimizer_params['multi_precision'] = True
-    opt = mx.optimizer.create(args.optimizer, sym=out, **optimizer_params)
+    opt = mx.optimizer.create('sgd', sym=out, **optimizer_params)
```

Comment:
why not let user pick optimizing algorithm?

Reply:
each optimizer has its own optimizer_params. very hard to support all of them